### DISCUSSION TITLE
UI Bug with trying to multi select

### DISCUSSION THREAD

**Discussion by austinwbest** [dani-garcia/vaultwarden#4339] [Q&A] [answered]
I installed this tonight for the first time via docker & i am sure this can be reproduced quite easily on more than the feb 1st release. Its quite frustrating to say the least.

I import lets say 1000 items and i want to organize them into collections or organizations. So i am scrolling through the list and checking one here and one there for awhile and then accidentally click on a row instead of the checkbox and the information modal pops up for that row... Welp everything i checked is now unchecked and need to start all over. After doing this a couple times i have now closed the UI and dont even want to bother with organizing it.

I am quite surprised this isnt already an open issue. Anyhow, its not a end of the world kinda thing but it is quite a frustrating thing that the UI does from a UX perspective.

**Comment by BlackDex** [comment, marked as answer]
First, thanks for creating A report.
Only, this project does north maintain the client interfaces.
That is probably also why there isn't a report here.

All client's are maintained by Bitwarden. We just use the web-vault with some small modifications.

I'm not sure which version you are running, but you might want to try the `testing` tagged images which have a newer version of the web-vaults.

**Reply by stefan0xC** [reply]
The bug has been reported https://github.com/bitwarden/clients/issues/5605 and https://github.com/bitwarden/clients/issues/7066 and there already is a PR in the work that should fix this behavior: https://github.com/bitwarden/clients/pull/6911

**Comment by austinwbest** [comment]
Gotcha, i misunderstood the project. I thought this was a rewrite but using the bitwarden engine on the backend. Thanks for linking those issues on their repository. Any further follow up with it can be done there!
